ArcadiaParties-SPA/src/app/routes.ts

```diff
@@ -1,7 +1,10 @@
 import { Routes } from '@angular/router';
 import { ContentComponent } from './content/content.component';
```

Comment:
Empty line needed between external and internal imports.

Reply:
Line added